staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/interface.go

```diff
@@ -45,7 +45,5 @@ type ValidatorCompiler interface {
 	BindingMatches(a admission.Attributes, o admission.ObjectInterfaces, definition *v1alpha1.ValidatingAdmissionPolicyBinding) (bool, error)
 
 	// Compile is used for the cel expression compilation
-	Compile(
-		policy *v1alpha1.ValidatingAdmissionPolicy,
-	) Validator
+	Compile(failPolicy *v1.FailurePolicyType, paramKind *v1alpha1.ParamKind, validations []v1alpha1.Validation) Validator
```

Comment:
I thought we couldn't import "k8s.io/pkg" into the staging directories

Reply:
Ah, good point. Using the existing webhook code as a guide, an accessor interface approach might be the way to handle this. Although, if it ends up just being a single `GetExpression` method, it might be simpler to just pass the expressions as a string slice instead...